docs-2.0/9.about-license/3.purchase-license.md

```diff
@@ -0,0 +1,48 @@
+# Purchase a Vesoft license 
+
+To utilize the features of the NebulaGraph database and associated software, you must obtain a Vesoft license. The license can be procured via the cloud marketplace or by directly [contacting Vesoft sales](https://www.nebula-graph.io/contact).
+
+Currently, AWS Marketplace is the only cloud marketplace where a license can be obtained. This article assists you in purchasing a Vesoft license on the AWS Marketplace.
+
+## Preparations
+
+You have registered an [AWS Marketplace](https://portal.aws.amazon.com/billing/signup#/start/email) account and logged in.
+
+## Steps
+
+1. Open the AWS Marketplace [NebulaGraph Enterprise (by Node)](https://aws.amazon.com/marketplace/pp/prodview-kvpxjh5b4dfno) service page.
+2. Click **View purchase options** to enter the license purchase contract signing page.
+3. Configure the contract items, including the license validity period, whether to enable auto-renewal, the number of query nodes, and the number of storage nodes.
+
+  - **How long do you want your contract to run**: Choose the validity period for the license, either 1 month or 1 year.
+  - **Renewal Settings**: Whether to automatically renew the license after its validity period. If you choose **Yes**, the license will be automatically renewed after its validity period. If you choose **No**, the license will not be automatically renewed after its validity period.
+  - **Contract Options**: Select the number of resources to purchase, currently supporting the purchase of query nodes and storage nodes.
+
+4. Click **Create contract**.
+5. In the pop-up panel, click **Pay now** to confirm the license purchase.
+6. Click **Click here to set up your account** to jump to the LC registration page to [set up your LC account](2.license-management-suite/2.license-center.md). 
+  You can view the license information on LC.
+
+  !!! note
+
+        After the registration is complete, you will receive an email from Vesoft within one working day. The email contains the NebulaGraph Enterprise Edition database and LM installation packages.
```

Comment:
Nebula Graph Enterprise packages instead of database only，because it includes other products such as NebulaGraph Explorer、Dashboard and etc.